### DISCUSSION TITLE
SetUnknownLanguage: Differentiate between Audio and Video

### DISCUSSION THREAD

**Discussion by minds3t** [ptr727/PlexCleaner#317] [Ideas]
I currently have _a lot_ of files for which the language tag of the video track is set to `und`. Some audio tracks are also unset. Most of the time setting `DefaultLanguage` to for example `de` would be sufficient for setting the audio track to the desired language. Gets complicated if it has multiple tracks in differerent languages but _undefined_. But that's not the point.

I just noticed that all video tracks also gets updated using the `DefaultLanguage` variable. Which is unpleasant. The videos are technically not always english nor german or whatever language is set in the settings. That's why I would like to differentiate between audio and video. I think the best solution would be to have an option to just ignore video ***or even better IMO*** an option to set the video tracks to `zxx` (No linguistic content) - Which is true in all cases as it is just video. This way we have a standardized tag for all video tracks.

**Comment by ptr727** [comment]
Content with multiple video tracks do have the video track language set, e.g. animation style movies where the background writing is in different languages.  
Majority of content have just one video track, so whatever the language is set to is not really important when it comes to player track selection, i.e. the player will select the only video track.  
`zxx` is as specific as `de`, implying there is no linguistic content in the video, which is rare as there is often some visible writing in a very specific language.

Video tracks could be excluded from setting a default language if `und`, but then configuration becomes more complicated when the video track language should be set, requiring more configuration options, so I opted for simplicity.
